### DISCUSSION TITLE
Install only packages not already installed

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#1781] [Linux] [answered]
I would like to know if there is an option to pass to `brew bundle` so that if there are some packages in the Brewfile already installed by other package managers those packages aren't installed by brew.

**Comment by SMillerDev** [comment, marked as answer]
No, brew does not have a concept of external packages. Something is either installed through brew or it's unavailable.

The reason for this is that there is no way to test if external dependencies actually supply everything brew needs.
